test/python/pulse/test_block_load_from_qpy.py

```diff
@@ -0,0 +1,193 @@
+# This code is part of Qiskit.
```

Comment:
While we're adding tests do you mind moving this to `test/python/qpy/test_schedule_block_from_qpy.py` mostly because I need to move the circuits tests over (so you're codeownership on qpy applies to it too).

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/7300/commits/a90a7d520f0506c0dfee22fa198ec64c29968a22